lib/src/main/java/com/auth0/android/lock/LockActivity.java

```diff
@@ -403,14 +431,16 @@ public void onDatabaseAuthenticationRequest(@NonNull DatabaseLoginEvent event) {
         AuthenticationAPIClient apiClient = options.getAuthenticationAPIClient();
         AuthenticationRequest request;
         Map<String, String> parameters = new HashMap<>(options.getAuthenticationParameters());
-        if (TextUtils.isEmpty(event.getMFAToken()) || TextUtils.isEmpty(event.getVerificationCode())) {
+        if (TextUtils.isEmpty(event.getMultifactorToken())) {
+            // regular database authentication
             String connection = configuration.getDatabaseConnection().getName();
             request = apiClient.login(event.getUsernameOrEmail(), event.getPassword(), connection);
-            if (!TextUtils.isEmpty(event.getVerificationCode())) {
-                parameters.put(KEY_VERIFICATION_CODE, event.getVerificationCode());
-            }
+        } else if (MFA_CHALLENGE_TYPE_OOB.equals(lastDatabaseLogin.getMultifactorChallengeType())) {
```

Comment:
if there's an `mfa_token` and also the challenge type is "oob", complete the verification using OOB